packages/capabilities/src/aggregate.js

```diff
@@ -44,17 +44,20 @@ export const offer = capability({
       /**
        * CID of the aggregate piece.
        */
-      link: Schema.link({
-        code: FilCommitmentUnsealed,
-        version: 1,
-        multihash: {
-          code: SHA2_256_TRUNC254_PADDED,
-        },
-      }),
+      link: /** @type {import('./types').PieceLinkSchema} */ (
```

Comment:
You can not because there is no way to verify it, so I was expecting that ppl would write manually type cast like `Schema<Link<T, ...>>` that way you you imply to know what you're doing.

I'm not sure what config param in ucanto would look like. That said I have in-progress PR that would allow us to infer types like `Schema.struct({ ... }).link({ codec, hasher, version })` which than would be able to infer the type from the schema.

Reply:
ok, will keep as is for the time being